include/MainWindow.h

```diff
@@ -81,9 +82,25 @@ class MainWindow : public QMainWindow
 	///
 	bool mayChangeProject(bool stopPlayback);
 
-	void autoSaveTimerStart()
+	// Auto save timer intervals
+	static const int DEFAULT_AUTO_SAVE_INTERVAL = 2;
+	int m_autoSaveInterval;
+	const static int m_autoSaveShortTime = 10 * 1000; // 10s short loop
+
+	void autoSaveTimerReset( int msec = ConfigManager::inst()->
+					value( "ui", "saveinterval" ).toInt()
+						* 60 * 1000 )
+	{
+		if( msec < 60 * 1000 || msec > 20 * 60 *1000 )
+		{
+			msec = DEFAULT_AUTO_SAVE_INTERVAL * 60 * 1000;
```

Comment:
Less than 1 is to capture 0 from users upgrading to 1.2 . More than 20 because I haven't tried out what happens if you manually edit the time.